### PR TITLE
chore: set up license-eye to check license header

### PR DESCRIPTION
**Pull request by kezhenxu94**
- [x] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #<issue number>. NO
- [x] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/CHANGES.md).

### CODE REVIEW

File: .github/actions/e2e-test/action.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        rm -rf skywalking-eyes
+        docker run --rm -v $(pwd):/github/workspace license-eye:latest header check
+        echo "::endgroup::"
+    - name: Build Docker Image
+      shell: bash
+      env:
+        SKIP_TEST: "true"
+      run: |
+        echo "::group::Build Docker Image"
+        make docker && ES_VERSION=es7 TAG=latest-es7 make docker.oap
```
Comment (wu-sheng) [inline .github/actions/e2e-test/action.yml:44]: Are we always using ES7?

**Review by wu-sheng — COMMENTED** [review]
Could we keep `Check License Header` in one task only? It seems repeatedly check the headers.

**Comment by kezhenxu94** [comment]
> Could we keep `Check License Header` in one task only? It seems repeatedly check the headers.

This is intentional, it serves as a barrier to fail fast when there is file missing license header.

File: .github/actions/e2e-test/action.yml (hunk 1 above)
Comment (kezhenxu94) [reply .github/actions/e2e-test/action.yml:44]: > Are we always using ES7?

Not necessary, if we already build a ES6 image, it costs only very little time to build ES7 too, so I wanted to make it simple

**Review by wu-sheng — COMMENTED** [review]
`skywalking-ui` submodule has been updated, please recheck. If this is your plan, please update the changes file.

**Comment by kezhenxu94** [comment]
> `skywalking-ui` submodule has been updated, please recheck. If this is your plan, please update the changes file.

Updated

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**